src/components/Modal/Account.vue

```diff
@@ -107,16 +107,22 @@ watch(open, () => (step.value = null));
             <Icon name="external-link" class="ml-1" />
           </UiButton>
         </a>
-        <UiButton @click="step = 'connect'" class="button-outline w-full mb-2">
-          {{ $t('connectWallet') }}
+      </div>
+    </div>
+    <template
+      v-slot:footer
+      v-if="step !== 'connect' && $auth.isAuthenticated.value"
+    >
+      <div class="w-2/4 float-left pr-2">
```

Comment:
Can you avoid float please. Use Flex wherever possible.

Reply:
Yeah I'll do that once this is less experimental